server/db/companies.js

```diff
@@ -25,3 +27,12 @@ function addCompany (company, db = knex) {
     .insert(company)
     .then(companyId => companyId[0])
 }
+
+function getProfile (id, db = knex) {
+  return db('roles').where(id = 'company_id')
+}
+
+function addProfile (profile, db = knex) {
+  return db('role_counts')
+    .insert(profile)
```

Comment:
probably need to convert from camelCase to train_case here

Reply:
I believe that's part of AddProfile branch